README.md

```diff
@@ -39,9 +39,11 @@ Using Airflow to implement our ETL pipelines
 1. Build docker image: `docker build -t davidtnfsh/pycon_etl:cache --cache-from davidtnfsh/pycon_etl:cache .`
 2. Start the Airflow server: `docker run  --rm -p 80:8080 --name airflow  -v $(pwd)/dags:/usr/local/airflow/dags -v $(pwd)/service-account.json:/usr/local/airflow/service-account.json davidtnfsh/pycon_etl:cache webserver`
     * service-account.json: Please contact @david30907d using email, telegram or discord.
-3. Setup the Authentication of GCP: <https://googleapis.dev/python/google-api-core/latest/auth.html>
+3. Fill out `airflow.cfg` with Google OAuth ID and credential (ref: <https://airflow.apache.org/docs/apache-airflow/1.10.1/security.html#setting-up-google-authentication>)
```

Comment:
I'd suggest against using bare URL. `Ref: [setting-up-google-authentication](https://airflow.apache.org/docs/apache-airflow/1.10.1/security.html#setting-up-google-authentication)` might be a better option.

Reply:
make sense to me~